app/views/items/new.html.erb

```diff
@@ -8,8 +8,12 @@
     <%= f.text_field :name, size: 60 %><br><br>
     <%= f.label :description %><br>
     <%= f.text_area :description, :cols => 60, :rows => 10 %><br><br>
-    <%= f.label :price %><br>
-    <%= f.text_field :price, size: 60 %><br><br>
+    <%= f.label :buy_it_now_price %><br>
+    <%= f.text_field :buy_it_now_price, size: 60 %><br><br>
```

Comment:
Rather than using so many <br> tags for spacing, you might want to use your css and just add some margin or padding.